packages/module-tenants/src/repository/TenantRecord.ts

```diff
@@ -0,0 +1,37 @@
+import type { TenantConfig } from '../models/TenantConfig'
```

Comment:
Maybe it's useful to add some kind of `usedAt` or `latestUsage` kind of property to indicate when the tenant was used for the last time. This might come in handy later on when doing a clean-up of old/unused tenants.

Reply:
Hmm ok, I get what you're saying. Maybe postpone this a bit indeed. It's useful but not a necessity either.